islands/collection/CollectionDetailsHeader.tsx

```diff
@@ -1,93 +1,77 @@
-import { useState } from "preact/hooks";
-
 export const CollectionDetailsHeader = (
-  { id, filterBy, sortBy }: { id: string; filterBy: any[]; sortBy: string },
+  { id }: { id: string },
 ) => {
-  const [selectedCategory, setSelectedCategory] = useState("BigTiles");
-
   return (
-    <div class="flex flex-col gap-3">
-      <div class="flex gap-4">
-        <img
-          src="/img/mock.png"
-          loading="lazy"
-          onError={(e) => {
-            e.currentTarget.src = `/not-available.png`;
-          }}
-          alt="collection image"
-          class="h-24 w-24 object-contain items-center standalone:h-24 standalone:w-auto pixelart image-rendering-pixelated"
-        />
-        <div class="flex flex-col justify-between">
-          <p class="font-semibold text-xl text-white">COLLECTION - {id}</p>
-          <div class="flex flex-col md:flex-row gap-4">
-            <div class="flex gap-4">
-              <div>
-                <p class="text-sm text-[#B9B9B9]">FLOOR</p>
-                <p class="text-lg font-semibold text-white">
-                  0.00018{" "}
-                  <span class="text-sm font-normal text-[#B9B9B9]">BTC</span>
-                </p>
-              </div>
-              <div>
-                <p class="text-sm text-[#B9B9B9]">TOTAL VOL</p>
-                <p class="text-lg font-semibold text-white">
-                  0.0019{" "}
-                  <span class="text-sm font-normal text-[#B9B9B9]">BTC</span>
-                </p>
-              </div>
-            </div>
-            <div class="flex gap-4">
-              <div>
-                <p class="text-sm text-[#B9B9B9]">OWNERS</p>
-                <p class="text-lg font-semibold text-white">1.1K</p>
-              </div>
-              <div>
-                <p class="text-sm text-[#B9B9B9]">LISTED</p>
-                <p class="text-lg font-semibold text-white">41</p>
-              </div>
+    <div className="flex flex-col gap-3">
+      <div className="dark-gradient p-6">
+        <div className="flex gap-4">
+          <img
+            src="/img/mock.png"
```

Comment:
same as below the getCollectionByName should be pulling the first image. If it does not there is an example on the home page island for these that pulls the first image in the collection.  (frequently the first image is broken so perhaps we should pull the 2nd image to be sure. for collections with only 1 image we can filter those out completely from displaying anywhere because those are typically broken